UPGRADING.md

```diff
@@ -1,6 +1,18 @@
 Upgrading
 =========
 
+## 6.x to 7.x
+
+### The new Exception interceptor
```

Comment:
I know `Exception` is a class name, but it irks me when I read it capitalised - in fact, we probably don't even need the h3 as it's the only noteworthy change.